converters/google/convert.go

```diff
@@ -249,10 +250,15 @@ func (c *Converter) augmentAsset(tfData converter.TerraformResourceData, cfg *co
 // projects. It implements a cache because many resources share the same
 // project.
 func (c *Converter) getAncestry(project string) (string, error) {
+	log.Printf("getAncestry: project=%s", project)
 	if path, ok := c.ancestryCache[project]; ok {
 		return path, nil
 	}
 
+	if c.resourceManager == nil {
```

Comment:
Since we allow nil resourceManager now, can we document this in `NewConverter()` and the `Converter` struct?